source/extensions/filters/http/cache/cache_headers_utils.h

```diff
@@ -54,12 +55,17 @@ struct RequestCacheControl {
   //   now - expiration_time < max-stale
   // If max-stale has no value then the client is willing to receive any stale response
   OptionalDuration max_stale_;
+
+private:
+  void applyCacheControlHeader(absl::string_view cache_control_header);
+  void applyPragmaHeader(absl::string_view pragma_header);
```

Comment:
Whoops, I think the changes to the .cc file were moved of to a different PR, so the .h changes probably should be too.

Reply:
Oops....